pybikes/utils.py

```diff
@@ -41,7 +41,6 @@ def sp_capwords(word):
 class PyBikesScraper(object):
     proxy_enabled = False
     last_request = None
-    ssl_verification = True
```

Comment:
The system didn't work locally without disabling ssl verification. I included it in the request call, please let me know if this should be done differently.

Reply:
You can set this without changing the class specification

```
foo = PyBikesScraper()
foo.ssl_verification = False
```

Reply:
But having it as an argument works for me too